### DISCUSSION TITLE
Dúvida CSS!

### DISCUSSION THREAD

**Discussion by P4dro-Dev** [BackandDeveloper/Projeto-Front-and-sobre-T-quio#2] [Q&A] [answered]
Como faço para deixar meu CSS mais organizado?

**Comment by iegoassis** [comment, marked as answer]
Aqui estão dicas para deixar seu CSS mais organizado:

Estrutura
1. *Ordem lógica*: Organize propriedades por tipo (layout, cor, fonte, etc.).
2. *Separe seções*: Utilize comentários para dividir o código em seções (header, footer, etc.).
3. *Crie um índice*: Adicione comentários para facilitar a navegação.

Nomeação
1. *Nomenclatura BEM*: Utilize bloco-elemento-modificador (ex.: `.header__logo--ativo`).
2. *Prefixos*: Utilize prefixos para classes (ex.: `.btn-`).
3. *Nomes descritivos*: Evite nomes genéricos.

Organização
1. *Arquivos separados*: Separe código em arquivos (ex.: `reset.css`, `layout.css`, `cores.css`).
2. *Importe ordenado*: Ordene imports por prioridade.
3. *Use variáveis*: Defina variáveis para cores, fontes, etc.

Formatação
1. *Indentação*: Utilize espaços ou tabs.
2. *Espaçamento*: Separe propriedades com uma linha em branco.
3. *Formatação automática*: Utilize ferramentas como Prettier.

Ferramentas
1. Prettier
2. CSSLint
3. Stylelint
4. Sass/Less
5. PostCSS

Boas práticas
1. *Mantenha atualizado*: Siga as últimas especificações do CSS.
2. *Teste*: Verifique compatibilidade entre navegadores.
3. *Documente*: Adicione comentários para explicar o código.
4. *Use pré-processadores*: Sass, Less ou Stylus.
5. *Minifique*: Reduza tamanho do arquivo.

Recursos adicionais
1. W3C ((link unavailable))
2. MDN Web Docs ((link unavailable))
3. CSS-Tricks ((link unavailable))
4. Smashing Magazine ((link unavailable))
5. Stack Overflow ((link unavailable))

Organizar seu CSS melhora:

- Legibilidade
- Manutenibilidade
- Desempenho
- Compatibilidade
- Reutilização de código
